deel-2-flexbox-layout/opdracht-1/styles/styles.css

```diff
@@ -1 +1,43 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+body{
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+
+main{
+    display: flex;
```

Comment:
Wellicht iets netter en voor de volgende keer overzichtelijker als je de flexbox property onder de item propeties zet. Main is namelijk een inner container van body dus krijgen de properties flex-basis, schrink en grow eerst en vervolgens kan main weer een eigen outer-container worden. + standaard is row-direction natuurlijk row maar mag inderdaad zelf bepalen of je het er neerzet of niet!